src/Command/QueueCleanupCommand.php

```diff
@@ -61,36 +55,27 @@ protected function configure(): void
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        $numberOfDays = self::DEFAULT_DAYS;
         // get the number of days from user
-        $numberOfDays = $input->getOption(self::DAYS_OPTION_NAME);
-        if (is_string($numberOfDays)) {
-            $numberOfDays = (int) $numberOfDays;
+        $numberOfDaysEntered = $input->getArgument(self::DAYS_ARGUMENT_NAME);
+        if (is_string($numberOfDaysEntered) && (int) $numberOfDaysEntered >= 0) {
+            $numberOfDays = (int)$numberOfDaysEntered;
         } else {
-            $numberOfDays = self::DEFAULT_DAYS;
+            $output->writeln('Sorry, the number of days entered is not valid!');
+            return self::FAILURE;
         }
 
         // get the beginning date
         $dateToDelete = $this->getPreviousDateNDays($numberOfDays);
 
         $queueItems = $this->queueItemRepository->findToCleanup($dateToDelete);
-        if (!$queueItems) {
-            $output->writeln('No items to delete found');
-
-            return self::FAILURE;
-        }
-
-        /** @var EntityManagerInterface $objectManager */
-        $objectManager = $this->managerRegistry->getManager();
 
         /** @var QueueItem $queueItem */
         foreach ($queueItems as $queueItem) {
-            $objectManager->remove($queueItem);
+            $this->queueItemRepository->remove($queueItem);
         }
 
-        // save DB edits
-        $objectManager->flush();
-
-        $output->writeln(sprintf('%c items deleted.', count($queueItems)));
+        $output->writeln(sprintf('%s items imported before %s deleted.', count($queueItems), $dateToDelete->format("Y-m-d H:i:s")));
```

Comment:
A better output should be "100 items imported before 2020-12-11 18:39:00 have been deleted.". You should also color items count and date with the Symfony console `<info>` style:

```php
$output->writeln(sprintf('<info>%s</info> items imported before <info>%s</info> has been deleted.', count($queueItems), $dateToDelete->format("Y-m-d H:i:s")));
```